cloud/azure/bin/lib/key_vault.sh

```diff
@@ -57,6 +57,21 @@ function key_vault::add_secret() {
     --value "${3}" > /dev/null
 }
 
+#######################################
+# Add a secret to the key vault
+# Arguments:
+#   1: The name of the key vault
+#   2: The name of the secret (used to identify it e.g. "postgres-password")
+#######################################
+function key_vault::add_secret_from_input() {
+  local SECRET
+  echo "Please enter the value for ${2}: "
+  read -s $SECRET && unset REPLY
```

Comment:
question (blocking): why `unset REPLY`?

Reply:
I was thinking we didn't want secret values hanging out in environment variables

Reply:
shubha explained to me in chat, this is a best practice